packaging/installer/README.md

```diff
@@ -202,6 +202,27 @@ _actively_ contributing to Netdata's future.
 
 We are tracking a few issues related to installation and packaging.
 
+#### Installs on hosts without IPv4 connectivity
+
+Our regular installation process requires access to a number of GitHub services that do not have IPv6 connectivity. As
+such, using the kickstart install script on such hosts generally does not work, and will typically fail with an
+error from cURL or wget about connection timeouts. You can check if your system is affected by this by attempting
+to connect to (or ping) `https://api.github.com/`. Failing to connect indicates that you are affected by this issue.
+
+There are three potential workarounds for this:
+
+1. You can configure your system with a proper IPv6 transition mechanism, such as NAT64. GitHub’s anacrhonisms
```

Comment:
```suggestion
1. You can configure your system with a proper IPv6 transition mechanism, such as NAT64. GitHub’s anachronisms
```

There is a typo here, but also "GitHub’s anacrhonisms" sounds a bit harsh to me. Maybe change it to "Github's practices" to sound less accusatory?

Reply:
I would normally agree, but IPv6 is twenty five years old at this point, people have been asking GitHub for full IPv6 connectivity for at least five years now with essentially no response from them, and on top of all of that the type of people likely to be affected by this will almost certainly agree with the sentiment.

Reply:
OK, I am convinced :) .